modules/core/src/scenegraph/gltf/gltf-environment.js

```diff
@@ -0,0 +1,97 @@
+import GL from '@luma.gl/constants';
+import {Texture2D, TextureCube} from '../../webgl';
+import {loadImage} from '../../core/load-file';
+
+export default class GLTFEnvironment {
+  constructor(gl, baseUrl) {
+    this.gl = gl;
+    this.baseUrl = baseUrl;
+  }
+
+  getTexUrl(envMap, type, dir, mipLevel = 0) {
+    return `${this.baseUrl}${envMap}/${type}/${type}_${dir}_${mipLevel}.jpg`;
+  }
+
+  getBrdfUrl() {
+    return `${this.baseUrl}brdfLUT.png`;
+  }
+
+  makeCube({id, getTextureForFace, parameters}) {
+    return new TextureCube(this.gl, {
+      id,
+      mipmaps: false,
+      parameters,
+      pixels: {
+        [GL.TEXTURE_CUBE_MAP_POSITIVE_X]: getTextureForFace('right'),
+        [GL.TEXTURE_CUBE_MAP_NEGATIVE_X]: getTextureForFace('left'),
+
+        [GL.TEXTURE_CUBE_MAP_POSITIVE_Y]: getTextureForFace('top'),
+        [GL.TEXTURE_CUBE_MAP_NEGATIVE_Y]: getTextureForFace('bottom'),
+
+        [GL.TEXTURE_CUBE_MAP_POSITIVE_Z]: getTextureForFace('front'),
+        [GL.TEXTURE_CUBE_MAP_NEGATIVE_Z]: getTextureForFace('back')
+      }
+    });
+  }
+
+  getDiffuseEnvSampler() {
+    if (!this._DiffuseEnvSampler) {
+      this._DiffuseEnvSampler = this.makeCube({
+        id: 'DiffuseEnvSampler',
+        getTextureForFace: dir => loadImage(this.getTexUrl('papermill', 'diffuse', dir)),
```

Comment:
papermill? This sounds specific to one such map? Replace with parameter?

Reply:
Adding it to base url.